src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -414,15 +414,16 @@ public Response useDefaultCitationDate(@Context ContainerRequestContext crc, @Pa
     @GET
```

Comment:
I'm just going to leave this comment on the first line of code...

Can you please merge the latest from develop?

Reply:
@johannes-darms me again. We just released 6.5. Can you please merge the latest from develop? Thanks.

Reply:
done